components/narrative-header/NarrativeHeaderEvent.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <header class="narrative-header-event">
+    <div class="narrative-header-event__header">
+      <h1 class="narrative-header-event__header-title">
+        {{ chapter.title }}
+      </h1>
+      <img
+        v-if="chapter.cover"
+        :src="chapter.cover.responsiveImage.src"
+        :srcset="chapter.cover.responsiveImage.srcSet"
+        :sizes="chapter.cover.responsiveImage.sizes"
+        alt=""
+        class="narrative-header-event__header-image"
+      >
+    </div>
+
+    <nav
+      v-if="chapter.pages.length > 1"
+      class="narrative-header-event__navigation"
+    >
+      <nuxt-link
+        :to="`/${this.$route.params.language}/events/${this.$route.params.event}`"
+        class="narrative-header-event__back-button"
+      >
+        Back
+      </nuxt-link>
+
+      <select
```

Comment:
Should we use the same component as in the regular NarrativeHeader?
![Screen Shot 2020-07-08 at 09 59 39](https://user-images.githubusercontent.com/1516059/86893364-cdcb4400-c101-11ea-8803-bd0d7137951b.jpg)